judgments/views/advanced_search.py

```diff
@@ -66,7 +66,7 @@ def advanced_search(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("GET requests only")
     else:
         form: AdvancedSearchForm = AdvancedSearchForm(request.GET)
-        params: dict = request.GET
+        params = request.GET
```

Comment:
did the typing cause a problem?

Reply:
Yep - params is not a dict (dict's don't have a function getitems)